generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
@pascalgrimaud Are you working on adding separate questions? If not then I can try that.  
Seems that not very hard to do, current code is:
```
    if (this.pipelines.includes('jenkins') || this.pipelines.includes('gitlab')) {
        choices.push({name: '[Docker] Perform the build in a docker container', value: 'docker'});
    }
```
Just separate to 2 different if clauses.